mobile/lib/interfaces/album_api.interface.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:immich_mobile/entities/album.entity.dart';
+
+abstract interface class IAlbumApiRepository {
```

Comment:
Ideally, this eventually is just `IAlbumRepository`, right?

Reply:
Not sure about that.

We now have
* 1 repository to access Albums in the local DB (AlbumRepo),
* 1 repository to access albums on the device via platform calls (AlbumMediaRepo)
* 1 repository to access albums on the server via open API (AlbumApiRepo)

Reply:
Right. The API one might go away depending on how the sync process works long term. Either way this send fine for now.